terraform/notification.alpha.canada.ca.tf

```diff
@@ -169,3 +169,43 @@ resource "aws_route53_record" "m-notification-alpha-canada-ca-NS" {
     ttl     = "30"
 
 }
+
+resource "aws_route53_record" "pbmm-notification-alpha-canada-ca-ACM-cname" {
+    zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+    name    = "_a07b492def227e59365aab830dfbe21a.alpha.notification.canada.ca"
+    type    = "CNAME"
+    records = [
+        "_f8291d5f10595f37e7a33d28c75bc10d.wggjkglgrm.acm-validations.aws."
+    ]
+    ttl     = "300"
+}
+
+resource "aws_route53_record" "pbmm-document-notification-alpha-canada-ca-ACM-cname" {
+    zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+    name    = "_0c4462819f8f2c25376092ba4e0eab1f.document.alpha.notification.canada.ca"
+    type    = "CNAME"
+    records = [
+        "_4dd0ef2252498893a7fdf8a8c5d968b9.wggjkglgrm.acm-validations.aws."
+    ]
+    ttl     = "300"
+}
+
+resource "aws_route53_record" "pbmm-notification-canada-ca-ACM-cname" {
+    zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
```

Comment:
This is not the correct zone. Zone should be form here https://github.com/cds-snc/dns/blob/master/terraform/notification.canada.ca-zone.tf#L1